### DISCUSSION TITLE
Command Logs / Item Receive, Drop Logs

### DISCUSSION THREAD

**Discussion by BabeElena** [Pryaxis/TShock#1740] [General]
I need to access logs for cheat / hack checking, have any way to access that?

**Comment by hakusaro** [comment]
Most events are logged to the log files in the actual server directory. Are you looking for an event or action that's not logged? Some antihack things I think we don't log because we just "fix them" but I think they're pretty rare. Can you give me a set of precise events you're looking for if you don't see them in your server's log files?
